invisible_cities/io/dst_io.py

```diff
@@ -24,3 +26,43 @@ def load_dst(filename, group, node):
 def load_dsts(dst_list, group, node):
     dsts = [load_dst(filename, group, node) for filename in dst_list]
     return pd.concat(dsts)
+
+class TableMismatchError(Exception):
+    def __init__(self):
+        s  = 'The table and dataframe dont match! '
+        Exception.__init__(self, s)
+
```

Comment:
This class should go in `core/exceptions.py`

Reply:
The other IC exceptions don't have a predefined message. Since it is used only in one place you may want to move the message to the place where the exception is actually raised.

Reply:
I moved it to exceptions.py and removed message, I think the name is self explanatory